pkgs/params/src/lib.zig

```diff
@@ -14,5 +14,5 @@ const activePresetValues = @field(presets, @tagName(activePreset));
 pub const SECONDS_PER_SLOT = activePresetValues.SECONDS_PER_SLOT;
 
 test "test preset loading" {
-    try std.testing.expect(SECONDS_PER_SLOT == 4);
+    try std.testing.expect(SECONDS_PER_SLOT == mainnetPreset.preset.SECONDS_PER_SLOT);
```

Comment:
@g11tech I also had to fix that, but please check if the choice makes sense

Reply:
reverted the slot time back, it was a debugging artifact, thanks for pointing :+1: :pray: